Added limitation for app only calls (#73621)
Azure/azure-content: articles/active-directory/external-identities/reset-redemption-status.md

3. Use one of the methods below to reset the user's redemption status.

> [!NOTE]
>During public preview, we have two recommendations:

> >- When you're resetting the user's email address to a new address, we recommend setting the `mail`. This way the user can redeem the invitation by signing into your directory in addition to using the redemption link in the invitation.

Reviewer: ```suggestion
>- When you're resetting the user's email address to a new address, we recommend setting the `mail` property. This way the user can redeem the invitation by signing into your directory in addition to using the redemption link in the invitation.
```

> >- When you are trying to reset the status on a B2B user, ensure you are doing this under user context. App only calls are currently not supported.

Reviewer: ```suggestion
>- When you're resetting the status for a B2B guest user, be sure to do so under the user context. App-only calls are currently not supported.
```
